build.boost.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+git submodule update --init --recursive ext/boost_1_82_0
```

Comment:
we add boost_1_82_0 submodule - branch/tag boost-1.82.0 per your established boost version.

NOTE: the brew build takes longer than this scripted build - I actually like building it this way better...